base/threaded_list.hpp

```diff
@@ -12,9 +12,28 @@ class ThreadedList : public ThreadedContainer
 private:
 
   std::list<T> m_list;
-  bool m_isEmpty;
-  std::string m_resName;
+  std::atomic<bool> m_isEmpty;
 
+  bool WaitNonEmpty(std::unique_lock<std::mutex> &lock)
+  {
+    bool doFirstWait = true;
+
+    while ((m_isEmpty = m_list.empty()))
+    {
+      if (IsCancelled())
+        break;
+
+      if (doFirstWait)
+        doFirstWait = false;
+
+      m_Cond.wait(lock);
```

Comment:
s/m_Cond/m_cond/g

Reply:
why? this is already existing data member, I do not invent this name.

Reply:
To fix invalid variable naming and make the code better and more consistent.